#Исключения
Исключения в RIDE выбрасываются с помощью функции `throw`:
```
func main(amount: Int) = {
   if (amount > 0) then "done" else throw()
}
```
Функция `throw` имеет возвращаемый тип [Nothing](/ride/data-types.md).

В RIDE нет обработки исключений. После того как исключение было выброшено, скрипт прекратит свое выполнение. Транзакция при этом будет считаться неудавшейся и не будет включена в блок.